native/rust/rspdk/src/main/rust/config/nvme.json

```diff
@@ -0,0 +1,17 @@
+{
+  "subsystems": [
+    {
+      "subsystem": "bdev",
+      "config": [
+        {
+          "method": "bdev_nvme_attach_controller",
+          "params": {
+            "name": "NVMe0",
+            "trtype": "PCIe",
+            "traddr": "0000:01:00.0"
```

Comment:
Add a note on the syntax here:
```
[[[[<domain>]:]<bus>]:][<slot>][.[<func>]]
```
The address may vary on different testing environments, and can be obtained by querying with `lspci -D`.